### DISCUSSION TITLE
Alguém já fez o curso do kentcdodds ?

### DISCUSSION THREAD

**Discussion by julianosirtori** [frontendbr/forum#2128] [Perguntas e Respostas]
Estou pensando em fazer o curso do kentcdodds, para o brasil tem um desconto de 60%, porem ainda o preço é meio salgado  hahhahaha

O link do curso é esse aqui:
https://epicreact.dev/

**Comment by fabiochiquezi** [comment]
Achei bem salgado o preço msm haha, até pq o que parece compensar mesmo é o "Epic React Pro"

**Reply by julianosirtori** [reply]
sim, para quem ganha em real fazer esses cursos em dolar acaba ficando bem caro mesmo hahhah

**Reply by ninetails** [reply]
Não deixa de ser um investimento né...

Mas tanto o curso dele quanto os dos Wes Bos geralmente tem um "desconto" pra brasileiros

Tem uns cursos do Kent no Egghead, um até gratuito se não me engano, bem massa.

**Reply by julianosirtori** [reply]
dahora @ninetails  vou dar uma olhada no Egghead

**Comment by nazarepiedady** [comment]
Eu tenho o curso de TDD em JavaScript dele, e vale a pena mesmo
